core/templates/dev/head/pages/landing/TopicLandingPage.ts

```diff
@@ -29,17 +31,61 @@ oppia.constant('TOPIC_LANDING_PAGE_DATA', {
     fractions: {
       collection_id: '4UgTQUc1tala',
       page_data: {
-        image_1: 'matthew_paper.png',
-        image_2: 'matthew_fractions.png',
+        image_1: {
+          file_name: 'matthew_paper.png',
+          alt: 'Matthew showing parts of fractions written on a card.'
+        },
+        image_2: {
+          file_name: 'matthew_fractions.png',
+          alt: 'Matthew solving problems on Oppia.'
+        },
         video: 'fractions_video.mp4',
+        lessons: [
+          'What is a Fraction?',
+          'Comparing Fractions',
+          'The Meaning of "Equal Parts"',
+          'Adding and Subtracting Fractions'
+        ]
+      }
+    },
+    'negative-numbers': {
+      collection_id: 'GdYIgsfRZwG7',
+      page_data: {
+        image_1: {
+          file_name: 'negative_1.png',
+          alt: 'A boy showing 3 + -24 written on a slate.'
+        },
+        image_2: {
+          file_name: 'negative_2.png',
+          alt: 'A boy smiling and solving negative-number problems on Oppia.'
+        },
+        video: 'negative-numbers_video.mp4',
+        lessons: [
+          'The number line',
+          'What is a negative number?',
```

Comment:
Use capitalized case: The Number Line, What is a Negative Number?, etc. -- for consistency.

Reply:
Done!